lemur/certificates/service.py

```diff
@@ -176,6 +176,22 @@ def get_all_certs_attached_to_endpoint_without_autorotate():
     )
 
 
+def get_all_certs_attached_to_destination_without_autorotate():
```

Comment:
what do you think about adding an optional destination `plugin_name`? this would allow for use-cases, where for instance only one type of destination should be auto-rotated. For instance, auto-rotate only `aws-s3`.
In case no `plugin_name` is provided, all destinations could be retrieved. 

the cli could then read the plugin_name from `ENABLE_AUTO_ROTATE_DESTINATION_TYPE`

Reply:
Perfect, added!